tests/karate/src/test/java/be/utils/GoServer.java

```diff
@@ -28,4 +28,11 @@ public String getDir() {
   public String getLogPath() {
     return Paths.get("go.log").toString();
   }
+
+  @Override
+  public void deleteDatabaseDir() {
+    // TODO Auto-generated method stub
+
```

Comment:
Maybe add a real comment here

```suggestion
    // The Go backend does not need to delete the database
```